tests/models/deta/test_modeling_deta.py

```diff
@@ -78,6 +81,7 @@ def __init__(
         self.hidden_dropout_prob = hidden_dropout_prob
         self.attention_probs_dropout_prob = attention_probs_dropout_prob
         self.num_queries = num_queries
+        self.two_stage_num_proposals = two_stage_num_proposals
```

Comment:
We should note that num_queries are not the final logits or pred_boxes shape, two_stage_num_proposals ㅁㄱㄷ